.github/workflows/github-pages.yml

```diff
@@ -35,6 +38,7 @@ jobs:
 
       - name: Build PWA 🛠️
         run: |
+          echo 'VITE_ROUTER_HISTORY_MODE="hash"' >> .env
```

Comment:
Move `VITE_ROUTER_HISTORY_MODE` below, under `env:` property. It should work without this hack.

Reply:
Also, you can remove `env` declaration on line 15.

https://github.com/Adamant-im/adamant-im/pull/778/files#diff-983e80aff18936dd64301171c8eec0d019a56ed0e2849985c5ba1c7cb48a8607R15-R17

Reply:
https://github.com/Adamant-im/adamant-im/pull/778/commits/054360bfd0f6c162a8b18f0ffac1542fd9b3a35f